docs/page.css

```diff
@@ -165,6 +169,32 @@ code.inline {
 	margin: 0;
 }
 
+.copy-btn {
+	cursor: pointer;
+	position: absolute;
+	top: 16px;
+	right: 16px;
+	width: 24px;
+	height: 24px;
+	background-color: transparent;
+	background-image: url('/files/ic_copy_grey_24dp.svg');
```

Comment:
Can you write these URLs like so:
```
background-image: url('../files/ic_copy_grey_24dp.svg');
```
That makes them more compatible with different hosting setups like githack. Check out the 404 in:

https://rawcdn.githack.com/snowiewdev/three.js/feature/copy-button/docs/index.html#manual/en/introduction/Creating-a-scene